week-4/D-arrays-of-objects/README.md

```diff
@@ -0,0 +1,29 @@
+### Goals
+
+- This is where the most complex exercises will go
+- A good time to do exercises involving arrays of objects. I.e. combining what we learned in the previous week + this one
+
+Stuff like this:
+
+```js
+var kitten1 = {
+    name: 'Fluffy',
+    weeksOld: 2
+};
+var kitten2 = {
+    name: 'Megatron',
+    weeksOld: 1
+};
+var kitten3 = {
+    name: 'Billy',
+    weeksOld: 5
+};
+
+var kittens = [kitten1, kitten2, kitten3];

```

Comment:
could you also have examples where you use the objects inside the array, i.e.

```js
var kittens = [{name: 'something'}, {name: 'something2'}]
```

Students seem to take things literally and really don't understand that these variables are labels that reference other stuff. That way also `json` is less of a shock for them.

Reply:
yes, good point